apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -95,28 +95,29 @@ export default class ChangeAction extends BaseRushAction {
     });
   }
 
-  public run(): void {
+  public run(): Promise<void> {
     console.log(`Target branch is ${this._targetBranch}`);
     this._projectHostMap = this._generateHostMap();
 
     if (this._verifyParameter.value) {
-      return this._verify();
+      this._verify();
+      return Promise.resolve();
     }
     this._sortedProjectList = this._getChangedPackageNames()
       .sort();
 
     if (this._sortedProjectList.length === 0) {
       console.log('No change file is needed.');
       this._warnUncommittedChanges();
-      return;
+      return Promise.resolve();
     }
 
     this._prompt = inquirer.createPromptModule();
     this._changeFileData = new Map<string, IChangeFile>();
     this._changeComments = ChangeFiles.getChangeComments(this._getChangeFiles());
 
     // We should consider making onExecute either be an async/await or have it return a promise
-    this._promptLoop()
+    return this._promptLoop()
       .catch((error: Error) => {
         console.error('There was an error creating the changefile:' + os.EOL + error.toString());
```

Comment:
>console [](start = 8, length = 7)

I think we want a nonzero exit code here, which means the error needs to propagate up to the top.  However, we also want the error to be reinterpreted here with extra contextual information.  So I think you should be throwing a new Error() with the improved description. 

Please validate that the console output for this situation is the same as before your change